tuxedo_sysfs/src/charging/mod.rs

```diff
@@ -0,0 +1,38 @@
+mod charge_control;
+mod charging_profile;
+
+// TODO: split ChargingProfile into Profile + Priority?
+// this would make the double Option<> thing for available + file less awkward to work with...
```

Comment:
Yes, this sounds good and should also solve the open questions in charging_profile.rs